packages/mocks/src/console-plugin/index.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import { rest, SetupWorker, setupWorker } from 'msw';
+
+type MockServiceWorkerContextType = {
+  worker: SetupWorker;
+};
+
+export const MockingContext = React.createContext<MockServiceWorkerContextType>({
+  worker: undefined,
+});
+
+export const MockingContextProvider = MockingContext.Provider;
+
+export const useValuesMockingContext = (): MockServiceWorkerContextType => {
```

Comment:
1. nice hack:)
2. do we need some code to actively call this hook? or is it enough that it's defined and loaded by the Console?

Reply:
It is enough that it is defined and loaded!

Also makes you think about what we could do with a "root context" and save data loading time between pages...